cli/lsp/language_server.rs

```diff
@@ -481,10 +481,11 @@ impl LanguageServer {
   }
 }
 
-fn create_npm_api_and_cache(
+pub fn create_npm_api_and_cache(
```

Comment:
This is a high level function used for construction and we shouldn't expose it to the rest of the application. We can instead find a way to break down the components to make them more usable in tests and revert the changes here.